modules/nf-core/windowmasker/main.nf

```diff
@@ -0,0 +1,44 @@
+process WINDOWMASKER {
+    tag "$meta.id"
+    label 'process_medium'
```

Comment:
Is this high of a label necessary for a module that runs on one thread?

Reply:
I assumed it was multi, done.